src/index.ts

```diff
@@ -431,26 +409,27 @@ export async function cli(args: string[]) {
 
   let isExplicit = false;
   const installTargets = [];
-  const {namedExports, webDependencies, dedupe} = pkgManifest['snowpack'] || {
-    namedExports: undefined,
-    webDependencies: undefined,
-    dedupe: undefined,
-  };
```

Comment:
These are now handled by cosmiconfig (because they may not be in `package.json`—they may be in `.snowpackrc`!)